### DISCUSSION TITLE
Wyzecam V3 RTSP Best Settings for Streaming Youtube

### DISCUSSION THREAD

**Discussion by PMFMAN** [gtxaspec/wz_mini_hacks#359] [Q&A]
Good Day Everyone.
I have a question about the potential best RTSP settings for setting up a WCV3 to record in OBS and then stream live out on the YouTube platform.
I have a Thermal Cutting Robot that I am setting up this WCV3 to record its operations live for clients/customers.  I get a few tearing here and there with the current settings below.  If any one has any suggestions that would be great.  Thank you in advance.  I am looking for best picture quality but also smoothest/highest frame rate for the live streaming.  This project is awesome and Hats off to the developers.

RTSP_HI_RES_ENABLED="true"
RTSP_HI_RES_ENABLE_AUDIO="true"
RTSP_HI_RES_FPS=""
RTSP_HI_RES_MAX_BITRATE="51200"
RTSP_HI_RES_TARGET_BITRATE="2048"
RTSP_HI_RES_ENC_PARAMETER="8"

**Comment by Bob-AOL** [comment]
RTMP settings are provided for that purpose.

RTMP Streaming:

```
RTMP_STREAM_ENABLED="true"
RTMP_STREAM_FEED="video1_unicast"
RMTP_STREAM_SERVICE="youtube"
RTMP_STREAM_DISABLE_AUDIO="false"
RTMP_STREAM_YOUTUBE_KEY="xxx-xxx-xxx-xxx"
RTMP_STREAM_TWITCH_KEY=""
RTMP_STREAM_FACEBOOK_KEY=""
```

Live stream DIRECTLY from the camera's local RTSP server to: `youtube` / `twitch` / `facebook` live.  Audio must be enabled in the RTSP section of the configuration for this to work.

**Reply by PMFMAN** [reply]
Bob thanks for your reply.   Below is a link to what I am trying to accomplish.  Today we tested it out and I got a bunch of tearing and stuttering.  Would reducing my bitrates above help with the tearing and stuttering?  If so what is the highest that can be used without losing quality?  Thanks!

https://www.youtube.com/watch?v=PiJu27dbsYU

**Comment by Bob-AOL** [comment]
Not an expert by any means, but would try this with audio enabled and see how it works. Basically the default settings (or recommended ones) from the initial .conf file. (I would not enable LO_RES settings.)

RTSP_HI_RES_ENABLED="true"
RTSP_HI_RES_ENABLE_AUDIO="false"
RTSP_HI_RES_FPS="20"
RTSP_HI_RES_MAX_BITRATE="2048"
RTSP_HI_RES_TARGET_BITRATE="1024"
RTSP_HI_RES_ENC_PARAMETER="2"

Have you tried the RTMP approach?

**Reply by PMFMAN** [reply]
Bob,
Thanks again for your reply!  I have not tried the rtmp approach yet.  I kind of like the OBS pass through method as I have a few other cameras I am playing with that I can remotely turn on and off from that software.  However, I am not ruling it out by any means.
I am having them pull the camera down so i can change to these settings on the chip.  I will see how these new parameters work hopefully better.  I think maybe the bitrate was just too high is all I am hoping.  I do have a strong wireless signal and an ac router so the wifi I do not believe is the problem for the video issues.
Thanks for your continued input!
